src/test/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpointIT.java

```diff
@@ -41,6 +42,7 @@
 public class CaseEndpointIT {
 
   private UUID collectionExerciseId;
+  private static HashMap<String, String> metadata;
```

Comment:
Does this need to be `static`?

Reply:
No it doesn't need to be static 👍

Reply:
This should also be a `Map` not a `HashMap`